core/shared/src/main/scala-2/cats/effect/cpsinternal/AsyncAwaitDsl.scala

```diff
@@ -98,7 +98,7 @@ object AsyncAwaitDsl {
             ${mark(q"""override def apply(tr$$async: _root_.cats.effect.cpsinternal.AsyncAwaitDsl.AwaitOutcome[$effect]): _root_.scala.Unit = $body""")}
           }
           ${F}.flatten {
-            _root_.cats.effect.std.Dispatcher[$effect].use { dispatcher =>
+            _root_.cats.effect.std.Dispatcher.parallel[$effect].use { dispatcher =>
```

Comment:
Shouldn't this be `sequential` since we will only use it within this block?

Reply:
Done.